### PR TITLE
Update lyrics-master to 2.4.5

### PR DESCRIPTION
**Pull request by shishi**
update lyrics-master version to 2.4.5

### CODE REVIEW

File: Casks/lyrics-master.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask :v1 => 'lyrics-master' do
-  version '2.4.4'
+  version '2.4.5'
   sha256 '659b958c8a4b9d3e24245398d0a08318b354d92815bbfd15d157cdc9367466c5'
```
Comment (scribblemaniac) [inline Casks/lyrics-master.rb]: You must update the checksum

**Comment by Ngrd** [comment]
Thank you for the contribution. It needed some fixes, so they were made in a85df95066c89510dedbd0bb245586785fd1da6a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by shishi** [comment]
Oh, I'm Sorry. I will fix needed changes.

**Comment by Ngrd** [comment]
No need. It is already fixed and merged and your contribution is still credited to you.

**Comment by shishi** [comment]
I have seen https://github.com/caskroom/homebrew-cask/commit/a85df95066c89510dedbd0bb245586785fd1da6a
I understand. Thank you for fixing!

**Pull request closed without merging**